portal/index.php

```diff
@@ -552,23 +552,23 @@ function enableVerifyBtn() {
             ?> <!-- Main logon -->
         <div class="container-xl p-1">
         <!-- Optionally show two logos, and in either order -->
-        <?php if($GLOBALS['secondary_portal_logo_position'] == 'second'){ ?>
-              <?php if ($GLOBALS['show_portal_primary_logo']) { ?>
-                <div class="img-fluid text-center"><img class="login-logo" src='<?php echo $logoSrc; ?>'></div>
-              <?php } ?>
-              <?php if ($GLOBALS['extra_portal_logo_login']) { ?>
-                <div class="img-fluid text-center"><img class="login-logo" src='<?php echo $logo2ndSrc; ?>'></div>
-              <?php } ?>
-        <?php } else {
-               if($GLOBALS['secondary_portal_logo_position'] == 'first'){ ?>
+            <?php if ($GLOBALS['secondary_portal_logo_position'] == 'second') { ?>
```

Comment:
You need to null coalesce most, especially new, variables to prevent warnings and eventually errors.
` if($GLOBALS['secondary_portal_logo_position'] ?? null == 'second'){`
always try to follow type such as if int `$x ?? 0` etc.

Reply:
thanks will make the change, that is good advice!